File: src/com/brianc/layout/AnonymousBlockBox.java
package com.brianc.layout;

import com.brianc.graphics.Renderer;

public class AnonymousBlockBox extends LayoutBox {
	@Override
	public BoxType getType() {
		return BoxType.ANONYMOUS_BLOCK;
	}

	@Override
	void layout(Dimensions containingBlock, Renderer renderBackend) {
	}
}

File: src/com/brianc/layout/AnonymousBlockBox.kt
package com.brianc.layout

import com.brianc.graphics.Renderer

class AnonymousBlockBox(override val children: MutableList<LayoutBox> = mutableListOf(),
                        override val dimensions: Dimensions = Dimensions()) : LayoutBox {
    override fun layout(containingBlock: Dimensions, renderBackend: Renderer) = Unit
}
